guides/plugins/plugins/administration/the-sanitizer-helper.md

```diff
@@ -0,0 +1,69 @@
+# The Sanitizer Helper
+
+## Overview
+
+The Shopware 6 Sanitizer Helper is a wrapper around [`DOMPurify`](https://github.com/cure53/DOMPurify), wich is used to sanitize HTML in order to prevent `XSS attacks`.
+
+## Where is it registered?
+
+The Sanitizer Helper is registered to the [Shopware Global Object](./the-shopware-object.md) and therefore can be accessed anywhere in your Plugin.
+
+```javascript
+const sanatizer = Shopware.Helper.SanitizerHelper; 
+```
+
+It also is registered in the Vue prototype as seen [here](https://github.com/shopware/platform/blob/trunk/src/Administration/Resources/app/administration/src/app/plugin/sanitize.plugin.js).
+This means it can also can accessed in your components like this:
+
+```javascript
+const Sanitizer = this.$sanitizer;
+const sanitize = this.$sanitize;
+```
+
+## Sanitizing HTML
+
+As mentioned before the SanitizerHelper is registered to the [Shopware Global Object](./the-shopware-object.md) and therefore can be accessed like this everywhere:
+```javascript
+Shopware.Helper.SanitizerHelper.sanitize('<img src=x onerror=alert(1)//>'); // becomes <img src="x">
+```
+
+And since it is bound to the Vue prototype it can be used in all Vue components like this:
+
+```javascript
+this.$sanitizer.sanitize('<svg><g/onload=alert(2)//<p>'); // becomes <svg><g></g></svg>
+this.$sanitize('<img src=x onerror=alert(1)//>'); // becomes <img src="x">
+```
+
+## How to set the config
+
+The config can be set with the and deleted with the function, as seen below:
```

Comment:
I'm not sure I understand this sentence. Maybe it's missing a word or there are too many? 🤔 
Probably: 

```suggestion
The config can be set deleted with the function, as seen below:
```